docs/references/remotes.md

```diff
@@ -0,0 +1,283 @@
+>        instead of the account key. If account key and sas token are both
+>        specified, account key will be used to sign. If any of account key, sas token
+>        or client_id are specified, anonymous access will be used.
+>
+>    - request_session: Session
+>        The session object to use for http requests.
+>
+>    - connection_string: str
+>        If specified, this will override all other parameters besides
+>        request session. See
+>        http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/
+>        for the connection string format.
+>
+>    - credential: azure.core.credentials_async.AsyncTokenCredential or SAS token
+>        The credentials with which to authenticate.  Optional if the account URL already has a SAS token.
+>        Can include an instance of TokenCredential class from azure.identity.aio.
+>
+>    - blocksize: int
+>        The block size to use for download/upload operations. Defaults to hardcoded value of
+>        ``BlockBlobService.MAX_BLOCK_SIZE``
+>
+>    - client_id: str
+>        Client ID to use when authenticating using an AD Service Principal client/secret.
+>
+>    - client_secret: str
+>        Client secret to use when authenticating using an AD Service Principal client/secret.
+>
+>    - tenant_id: str
+>        Tenant ID to use when authenticating using an AD Service Principal client/secret.
+>
+>    - anon: boolean, optional
+>        The value to use for whether to attempt anonymous access if no other credential is
+>        passed. By default (``None``), the ``AZURE_STORAGE_ANON`` environment variable is
+>        checked. False values (``false``, ``0``, ``f``) will resolve to `False` and
+>        anonymous access will not be attempted. Otherwise the value for ``anon`` resolves
+>        to ``True``.
+>
+>    - default_fill_cache: bool = True
+>        Whether to use cache filling with open by default
+>
+>    - default_cache_type: string ('bytes')
+>        If given, the default cache_type value used for "open()".  Set to none if no caching
+>        is desired.  Docs in fsspec
+>
+>    - version_aware : bool (False)
+>        Whether to support blob versioning.  If enable this will require the
+>        user to have the necessary permissions for dealing with versioned blobs.
+>
+>    - assume_container_exists: Optional[bool] (None)
+>        Set this to true to not check for existence of containers at all, assuming they exist.
+>        None (default) means to warn in case of a failure when checking for existence of a container
+>        False throws if retrieving container properties fails, which might happen if your
+>        authentication is only valid at the storage container level, and not the
+>        storage account level.
+>
+>    - max_concurrency:
+>        The number of concurrent connections to use when uploading or downloading a blob.
+>        If None it will be inferred from fsspec.asyn._get_batch_size().
+>
```

Comment:
```suggestion
- `max_concurrency`: `int` (default: `None`)

    The number of concurrent connections to use when uploading or downloading a blob.
    If `None` it will be inferred from `fsspec.asyn._get_batch_size()`.

```